ray-operator/config/samples/ray-cluster.mini.yaml

```diff
@@ -64,6 +64,15 @@ spec:
             name: dashboard
           - containerPort: 10001
             name: client
-
-
-
+          volumeMounts:
```

Comment:
Let's keep the `mini` one clean and easy to launch the cluster without additional changes?

Reply:
Sure ~~